tensorboard/uploader/formatters_test.py

```diff
@@ -19,6 +19,15 @@
 from __future__ import division
 from __future__ import print_function
 
+import os
+import time
+
+try:
+    # python version >= 3.3
+    from unittest import mock
+except ImportError:
+    import mock  # pylint: disable=unused-import
```

Comment:
All our unit tests run exclusively in Python 3. Please just import
`unittest.mock`. (We’d like to remove the `mock` backport. It has lots
of problems.)

Reply:
Done for this file. There are a few instances of this importing pattern in other parts of the tensorboard/uploader code. That can be fixed in a separate PR.

Reply:
Yep, of course—no imminent need to update existing code, just would
prefer to avoid introducing it in new code.